### ISSUE TITLE
Server log gets spammed after removing Eldritch Mobs

### ISSUE THREAD

**Issue by FoxySophie** [HyperPigeon/Eldritch-Mobs#99]
We used Eldritch Mobs for a while in our custom modpack but decided to remove it for now because most of the players didn't really like it. Now after removing the mod from the pack I noticed that the server log is getting spammed with the following messages:

`Failed to deserialize component: unregistered key eldritch_mobs:eldritch_modifiers`
And
`[Server thread/ERROR]: No codec for memory: minecraft:dummy; No codec for memory: minecraft:dummy; No codec for memory: minecraft:dummy; No codec for memory: minecraft:dummy`

I'm not sure if that second one has anything to do with it, but I'd like to a least fix the first one. Do I need to remove something else or is that because there are entities of the mod left?
This is on Minecraft 1.20.1 and we used the latest version as of writing this for that version.

**Comment by Osrock626** [comment]
Yeah i'm getting the same issue, same reason of removal due to players already overcoming the challenge presented with this mod. Good mod, but I needed to shed some weight of my server. Now I get spammed in the log with: 

[18:43:14] [Server thread/WARN]: Failed to deserialize component: unregistered key eldritch_mobs:eldritch_modifiers

I tried already searching through the files of the server, and removed the config but im still unsure what to do.
